lib/Feature/FeatureModelParser.cpp

```diff
@@ -221,7 +255,11 @@ std::unique_ptr<xmlDoc, void (*)(xmlDocPtr)> FeatureModelXmlParser::parseDoc() {
   if (Doc && Ctxt->valid) {
     xmlValidateDtd(&Ctxt->vctxt, Doc.get(), createDtd().get());
     if (Ctxt->vctxt.valid) {
-      return Doc;
+      if (xmlValidateDtdFinal(&Ctxt->vctxt, Doc.get()) == 1) {
+        return Doc;
+      }
+      llvm_unreachable("boink");
+      std::cerr << "Failed to validate DTD in final step" << std::endl;
```

Comment:
Try to use `llvm::errs()`.
This currently is a little bit inconsistent in the feature lib, I'll fix this in the future.
See se-passau/VaRA#713